### ISSUE TITLE
Cannot import pytorch to jupyter notebook

### ISSUE THREAD

**Issue by mattpodolak** [pytorch/pytorch#59555]
## 🐛 Bug

Cannot import pytorch to jupyter notebook, when running `import torch` the kernel dies immediately.

## To Reproduce

1. conda install pytorch torchvision torchaudio cudatoolkit=11.1 -c pytorch -c conda-forge
2. import torch

## Expected behavior

I expected it to not crash

## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0): pytorch-1.8.1 
 - OS (e.g., Linux): Windows 10
 - How you installed PyTorch (`conda`, `pip`, source): conda
 - Build command you used (if compiling from source): included above
 - Python version: 3.6
 - CUDA/cuDNN version: 11.1
 - GPU models and configuration:
 - Any other relevant information:

## Additional context
jupyter-notebook : 6.4.0

cc @peterjc123 @mszhanyi @skyline75489 @nbcsm

**Comment by skyline75489** [comment]
What exactly is the error? You basically gives us nothing to work with.

**Comment by mattpodolak** [comment]
@skyline75489 Unfortunately there was no error message printed out by Jupyter Notebook, the kernel just died. At the time of creating this issue I hadn't uncovered a fix, but I think I can provide some more insights now that this has been resolved.

I had just installed pytorch using the command described above, while already having a non-CUDA version of pytorch already installed on my machine (not sure the version). This non-CUDA version was being used in another Jupyter Notebook that was running. When I created a new notebook and tried to import, the kernel crashed.

I uninstalled pytorch using pip, and an error was thrown: `PermissionError: [WinError 5] Access is denied: <path_to_site-packages>\~orch\\lib\\asmjit.dll.conda_trash'`. I forced shutdown the jupyter notebook that was running, and fully uninstalled and re-installed pytorch, this time with pip. I can now import pytorch into Jupyter Notebook again.
